vaccine_feed_ingest/runners/co/colorado_gov/normalize.py

```diff
@@ -0,0 +1,296 @@
+            schema.Contact(
+                contact_type=schema.ContactType.GENERAL,
+                other="\n".join(site["description"]),
+            )
+        )
+
+    if len(contacts) == 0:
+        # Try to guess the website for major brands
+        website = _get_provider_store_page(site)
+        if website:
+            contacts.append(
+                schema.Contact(contact_type=schema.ContactType.GENERAL, website=website)
+            )
+
+    if len(contacts) > 0:
+        return contacts
+
+    return None
+
+
+def _get_inventory(site: dict) -> Optional[List[schema.Vaccine]]:
+    vaccines = []
+
+    if "ExtendedData" in site:
+        have_moderna = site["_folder_name"] == "Moderna"
+        have_pfizer = site["_folder_name"] == "Pfizer"
+        for field in ["Description", "Mapped Description", "unnamed (1)"]:
+            if field in site["ExtendedData"]:
+                if "moderna" in site["ExtendedData"][field].lower():
+                    have_moderna = True
+                if "pfizer" in site["ExtendedData"][field].lower():
+                    have_pfizer = True
+        if have_moderna:
+            vaccines.append(schema.Vaccine(vaccine=schema.VaccineType.MODERNA))
+        if have_pfizer:
+            vaccines.append(schema.Vaccine(vaccine=schema.VaccineType.PFIZER_BIONTECH))
+
+    if len(vaccines) > 0:
+        return vaccines
+
+    return None
+
+
+def _get_parent_organization(site: dict) -> Optional[schema.Organization]:
+    maybe_provider = provider_id_from_name(site["name"])
+    if maybe_provider:
+        return schema.Organization(id=maybe_provider[0])
+
+    return None
+
+
+def _get_links(site: dict) -> Optional[List[schema.Link]]:
+    maybe_provider = provider_id_from_name(site["name"])
+    if maybe_provider:
+        return [schema.Link(authority=maybe_provider[0], id=maybe_provider[1])]
+
+    return None
+
+
+def _get_notes(site: dict) -> Optional[List[str]]:
+    notes = []
+
+    if "ExtendedData" in site:
+        if (
+            "registrtiondescription" in site["ExtendedData"]
+            and site["ExtendedData"]["registrtiondescription"]
+            and site["ExtendedData"]["registrtiondescription"] != "#N/A"
+        ):
+            notes.append(site["ExtendedData"]["registrtiondescription"])
+        if (
+            "spanishresources" in site["ExtendedData"]
+            and site["ExtendedData"]["spanishresources"]
+            and site["ExtendedData"]["spanishresources"] != "#N/A"
+        ):
+            notes.append("Spanish: " + site["ExtendedData"]["spanishresources"])
+
+    if len(notes) > 0:
+        return notes
+
+    return None
+
+
+def _get_normalized_location(
+    site: dict, timestamp: str
+) -> Optional[schema.NormalizedLocation]:
+    source = "co_colorado_gov"
+    id = _get_id(site)
+
+    contacts = _get_contacts(site)
+    if not contacts:
+        logger.warning("No contacts for %s %s", site["name"], site["address"])
+        return None
```

Comment:
Please include the sites _even if_ they have no `Contacts`. Part of the goal of this pipeline is to merge available data sources which have different portions of information - it's possible that some other source will have contact info for these sites, but might not have the same notes, or similar.

Reply:
Whoops, I mistakenly believed that null contacts were forbidden by the schema. Fixed!